core/module/instance_methods_spec.rb

```diff
@@ -6,6 +6,11 @@
     methods = ModuleSpecs::Parent.instance_methods(false)
     methods.should_not include(:undefed_method)
   end
+  
+  it "does not consider methods as superclass methods if invoked on the parent module" do
```

Comment:
Opal was having an issue where when a module was included in a class, etc., you could no longer retrieve the instance methods on the base module. This is trying to ensure you can still do that.